platform/ui/src/components/downloadDialog/DownloadDialog.js

```diff
@@ -0,0 +1,235 @@
+import './DownloadDialog.styl';
+
+import React, { PureComponent } from 'react';
+import { withTranslation } from '../../utils/LanguageProvider';
+import Modal from 'react-bootstrap-modal';
```

Comment:
I think this is fine for now, but I'm going to create a different issue. We shouldn't be creating _many_ modals that use this dependency. We should have one, and then each other modal is simply content that can be the _child_ for the modal component, and has the ability to supply _actions_ to influence buttons (or something like that)

Reply:
Indeed, it deserves a new issue.